### DISCUSSION TITLE
Tokio logo usage on my site

### DISCUSSION THREAD

**Discussion by DariuszOstolski** [tokio-rs/tokio#4260] [General] [answered]
Hi,

I wrote a server that is using Tokio runtime. I have a question, may I use Tokio project logo in following context?

![obraz](https://user-images.githubusercontent.com/4119217/142777207-b46d680b-38e7-495c-9883-ebd80f68f5a6.png)

BR,
Dariusz

**Comment by carllerche** [comment, marked as answer]
Thanks for asking. For tokio, follow the same guidelines as Rust: https://www.rust-lang.org/policies/media-guide

If sounds like it is fine as you are stating your project uses Tokio.
